docs/datarepo.rst

```diff
@@ -139,6 +139,36 @@ Adds the dataset with the name ``1kg`` and description
 registry database ``registry.db``.
 
 
+----------------------
+add-dataset-duo
+----------------------
+
+Create/update new Data Use Ontology Information for an existing dataset. Note that you have to
+have an existing dataset to be able to use this command. When you need to update the DUO info,
+simply run the command with updated DUO Json file.
+
+Please note that as of now, you need to have an active internet connection to be able to use this,
```

Comment:
Can this be a problem with a container setup? Or the http port is usually open?

Reply:
That's right and it is more than likely possible. Good.

Reply:
Actually - when we revisited the newly implemented white-list DUO ids, this is no longer necesssary. This is in place previously because I wanted to allow any ID that is not on the blacklist to be validated against the DUO id pool.

Now that we do a whitelist, there's no need to validate them against the DUO id pool